manifests/j/JetBrains/Writerside/2023.2/JetBrains.Writerside.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: JetBrains.Writerside
+PackageVersion: '2023.2'
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: uninstallPrevious
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
`x86` validations are performed on 32-bit OSes.